docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -0,0 +1,37 @@
+# Grid Layout
+
+## Overview
+
+`GridLayout` is a grid box for the two-dimensional layout. It constraints the x and y position, width, and height of the child actors.
+
+Positioning children in a grid form, the cells are of uniform size based on the first child added to the parent View.
+
+The number of **columns** can be specified and the rows will automatically increased to hold the children. Once the available space is used up, further rows will not be added.
+
+| Property               | Type            | Description |
+| -----------------------| --------------- | ------------ |
+| `Columns`              | int             | Gets/Sets the number of columns in the Grid |
```

Comment:
To maintain consistency with the files in this PR, I am not using a period (.) at the end of a sentence in a table description. However, in other Tizen files, many places periods (.) are used.

@mijin85cho 
Please share your valuable thought on how to proceed.